BREAKING.md

```diff
@@ -12,10 +12,14 @@ There are a few steps you can take to write a good change note and avoid needing
 
 ## 0.49 Breaking changes
 - [Deprecated dirty document events and property removed from ContainerRuntime](#deprecated-dirty-document-events-and-property-removed-from-containerruntime)
+- [DeltaManager removed from fluid-framework export](#deltamanager-removed-from-fluid-framework-export)
 
 ### Deprecated dirty document events and property removed from ContainerRuntime
 The `isDocumentDirty()` method, `"dirtyDocument"` and `"savedDocument"` events that were deprecated in 0.35 have now been removed.  For more information on replacements, see [DirtyDocument events and property](#DirtyDocument-events-and-property).
 
+### DeltaManager removed from fluid-framework export
+The `DeltaManager` class has been removed from the `@fluid-framework/container-loader` package exports. `DeltaManager` is to be only used within the `@fluid-framework/container-loader` package.
```

Comment:
Perhaps remove the `DeltaManager is to be only used..` as this document is targeted primarily at the framework consumers. I would add a note saying something like the IDeltaManager interface should be used as appropriate.